packages/webpack/src/builder.js

```diff
@@ -185,6 +185,18 @@ export class WebpackBundler {
         })
     )
 
+    // disable webpack-dev-middleware stas printing
```

Comment:
Shall we have a comment about this line linking to PR?

Reply:
I found we're not configuring `stats`, so I changed the fix. 
Does this make sense to you ?

Reply:
BTW, should we remove [stats in webpackbar options ](https://github.com/nuxt/nuxt.js/blob/b4663d2bfb03237a429b7fa348770c831ed5e744/packages/webpack/src/config/base.js#L448)? I can't find any stats option usage in wepbackbar